ProjectSettings/ProjectSettings.asset

```diff
@@ -3,7 +3,7 @@
 --- !u!129 &1
```

Comment:
Don't commit your project settings. I'd revert the commit and remove this from being included.